hera_cal/abscal_funcs.py

```diff
@@ -643,10 +757,11 @@ def array_axis_to_data_key(data, array_index, array_keys, key_index=-1, copy_dic
         return new_data
 
 
-def UVData2AbsCalDict(filenames, pol_select=None, pop_autos=True, return_meta=False):
+def UVData2AbsCalDict(filenames, pol_select=None, pop_autos=True, return_meta=False, filetype='miriad',
+                        pick_data_ants=True):
     """
     turn pyuvdata.UVData objects or miriad filenames 
-    into the dictionary form that AbsCal requires. This format is
```

Comment:
`filenames` is a confusing command line argument, since you take uvdata objects and also (lists of) miriad or uvfits files.